{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-13299",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2020/CVE-2020-13299.json",
                    "name": "https://gitlab.com/gitlab-org/cves/-/blob/master/2020/CVE-2020-13299.json",
                    "refsource": "CONFIRM",
                    "tags": []
                },
                {
                    "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/222508",
                    "name": "https://gitlab.com/gitlab-org/gitlab/-/issues/222508",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https://hackerone.com/reports/896225",
                    "name": "https://hackerone.com/reports/896225",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "A vulnerability was discovered in GitLab versions before 13.1.10, 13.2.8 and 13.3.4. The revocation feature was not revoking all session tokens and one could re-use it to obtain a valid session."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-09-14T19:15Z",
    "lastModifiedDate": "2020-09-14T19:32Z"
}